stats/src/main/java/io/airlift/stats/QuantileDigest.java

```diff
@@ -882,6 +972,39 @@ private boolean postOrderTraversal(Node node, Callback callback, TraversalOrder
         return callback.process(node);
     }
 
+    protected boolean inOrderTraversal(Node node, Callback callback, TraversalOrder order)
```

Comment:
This should be private. This class is not meant to be extended, so there's not benefit in making it protected.